<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorTab</class>
 <widget class="QWidget" name="MotorTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Key">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Key Motor</string>
   </property>
   <widget class="QCheckBox" name="alive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Alive</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="setCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="busVoltage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="busCurrentLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bus Current</string>
    </property>
   </widget>
   <widget class="QLabel" name="busVoltageLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bus Voltage</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="setVelocity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="vehicleVelocityLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Vehicle Velocity</string>
    </property>
   </widget>
   <widget class="QLabel" name="setCurrentLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Current</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="busCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vehicleVelocity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="setVelocityLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Velocity</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="DetailsGroup">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>621</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>Motor Details</string>
   </property>
   <widget class="QDoubleSpinBox" name="phaseCCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>11.109999999999999</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="motorCurrentImaginary">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>14.140000000000001</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="motorVoltageImaginary">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>14.410000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="backEMF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>15.510000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="voltageRailSupply15V">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>17.710000000000001</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="motorCurrentReal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>13.130000000000001</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="motorVoltageReal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>13.310000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="phaseBCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>12.210000000000001</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Phase C Current</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Phase B Current</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor Voltage Real</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor Voltage Imaginary</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor Current Real</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor Current Imaginary</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Back EMF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>15V Voltage Rail Supply</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="voltageRailSupply3V">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>18.809999999999999</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="heatSinkTemp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>20.020000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="voltageRailSupply1V">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>19.910000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspBoardTemp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>22.219999999999999</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="motorTemp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>21.120000000000001</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="odometer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>24.420000000000002</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dcBusAmphours">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>23.320000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="slipSpeed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <double>25.520000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3.3V Voltage Rail Supply</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1.9V Voltage Rail Supply</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_46">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Heat-sink Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_47">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_48">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DSP Board Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_49">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DC Bus Amphours</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_50">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Odometer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_51">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Slip Speed</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
